cwl/src/main/scala/cwl/WorkflowStep.scala

```diff
@@ -67,7 +66,7 @@ case class WorkflowStep(
       // Create a task definition for the underlying run.
       // For sub workflows, we'll need to handle the case where this could be a workflow definition
       //TODO: turn this select into a fold that supports other types of runnables
-      val taskDefinition = run.select[CommandLineTool].map { _.taskDefinition } get
+      val taskDefinition = run.select[CommandLineTool].map { _.buildTaskDefinition(Option(workflow), _.validNel) } get
```

Comment:
The `_.validNel` is a particularly easygoing implementation of a `Requirement` validator. This is just a placeholder to keep what everyone else is working on from breaking, unless you're working on the `ex:BlibberBlubberFakeRequirement` conformance test.